impressions/local_static/css/application.css

```diff
@@ -389,27 +389,22 @@ picture {
   margin: 0;
   max-width: 100%; }
 
-.digdeeper {
-  height: 40vh;
-  margin: 0 0 1.6em 0; }
-  .digdeeper li {
-    display: inline-block;
-    margin-right: 1.6em; }
-
-.top-row li, .bottom-row li {
-  display: inline-block;
-  margin-right: 1.6em; }
-  .top-row li img, .bottom-row li img {
+.deeper .table-cell {
+  padding: 0 1em;
+  /* this adds left & right padding to images in deeper gallery */
+  height: calc(50vh - 1rem);
+  /* shouldn't need this once it's in neat environment */ }
+  .deeper .table-cell img {
     border: 1px solid #b3aa98;
+    width: 100%;
```

Comment:
instead add `max-width: none;` to counter act the max-width here: https://github.com/knowyourider/imp_project/pull/2/files#diff-2d514bfe0231ce2af90bb14f3902f0cfR390